apps/central-api/src/shared/helpers/ValidationHelper.ts

```
@@ -0,0 +1,13 @@
+import { GqlValidationEx } from '@shared/exceptions'
+import { z } from 'zod/v4'
+
+export class ValidationHelper {
+  static validate<T>(object: any, schema: z.ZodType): T {
+    const result = schema.safeParse(object)
+    if (result.error) {
+      throw new GqlValidationEx(z.prettifyError(result.error), result.error.issues)
+    }
+    return result as T
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)